docs/deploying-cf-locally.md

```diff
@@ -0,0 +1,348 @@
+-o ~/workspace/bosh-deployment/virtualbox/cpi.yml \
+-o ~/workspace/bosh-deployment/virtualbox/outbound-network.yml \
+-o ~/workspace/bosh-deployment/bosh-lite.yml \
+-o ~/workspace/bosh-deployment/bosh-lite-runc.yml \
+-o ~/workspace/bosh-deployment/uaa.yml \
+-o ~/workspace/bosh-deployment/credhub.yml \
+-o ~/workspace/bosh-deployment/jumpbox-user.yml \
+--vars-store ./creds.yml \
+-v director_name=bosh-lite \
+-v internal_ip=192.168.56.6 \
+-v internal_gw=192.168.56.1 \
+-v internal_cidr=192.168.56.0/24 \
+-v outbound_network_name=NatNetwork
+```
+Check that the VMs have been created in VirtualBox:
+
+```bash
+VBoxManage list vms
+```
+
+You should see entries similar to:
+
+```bash
+"sc-7ff79fac-045c-46e2-437f-ba09d54c40bd" {ca1d6ba0-e95d-4849-9d0f-f7723386951e}
+"vm-56718394-d4c1-4b94-6ce3-3d22c712844b" {56718394-d4c1-4b94-6ce3-3d22c712844b}
+```
+
+## Configure environment variables and login
+
+```bash
+export BOSH_CLIENT=admin
+export BOSH_CLIENT_SECRET=`bosh int ./creds.yml --path /admin_password`
+bosh alias-env vbox -e 192.168.56.6 --ca-cert <(bosh int ./creds.yml --path /director_ssl/ca)
+```
+
+Expected output:
+
+```bash
+Using environment '192.168.56.6' as client 'admin'
+
+Name               bosh-lite  
+UUID               d0207431-94e8-416e-a8ef-73f007d171c4  
+Version            282.0.4 (00000000)  
+Director Stemcell  -/1.822  
+CPI                warden_cpi  
+Features           config_server: enabled  
+                   local_dns: enabled  
+                   snapshots: disabled  
+User               admin  
+
+Succeeded
+helios02 :: 
+```
+## Set up routing for VM access
+Optionally, set up a local route for bosh ssh commands or accessing VMs
+directly:
+
+`sudo ip route add 10.244.0.0/16 via 192.168.56.6 # Linux (using iproute2 suite)`
+
+try ping 
+`ping 192.168.56.6`
+
+then try 
+`bosh -e vbox env`
+
+They should succeed both.
+
+## Upload cloud config and stemcell
+<!--
+the cloud config tells the BOSH Director how to provision VMs, networks, disks, and other IaaS-specific resources in your environment. -->
+
+```bash
+bosh -e vbox update-cloud-config ~/cf-deployment/iaas-support/bosh-lite/cloud-config.yml
+```
+
+```bash
+bosh -e vbox upload-stemcell \
+  --sha1 4ad3b7265af38de84d83887bf334193259a59981 \
+  "https://bosh.io/d/stemcells/bosh-warden-boshlite-ubuntu-jammy-go_agent?v=1.423"
+```
+
+## Clone the cf-deployment repository
+
+
+
+
+```bash
+git clone https://github.com/cloudfoundry/cf-deployment.git
+cd cf-deployment
+```
+
+Update the deployment file to use precompiled releases:
+
+
+```bash
+yq e '.stemcells[0].alias = "default" | .stemcells[0].os = "ubuntu-jammy" | .stemcells[0].version = "1.423"' -i cf-deployment.yaml
+```
+
+## Deploy Cloud Foundry
+```bash
+bosh -n -e vbox -d cf deploy \
+  cf-deployment.yml \
+  -o operations/bosh-lite.yml \
+  -o operations/use-compiled-releases.yml \
+  -v system_domain=bosh-lite.com \
+  -v stemcell_os=ubuntu-jammy \
+  -v stemcell_version=1.423
```

Comment:
I'd export this value as an env variable in case this changes in the future due to new versions. WDYT?